### ISSUE TITLE
Lack of scroll bars on the left menu and main window

### ISSUE THREAD

**Issue by derrickmehaffy** [strapi/strapi#3904]
**Describe the bug**
For accessibility purposes there should be scroll bars for the left menu and the main window as not everyone has the option for touchpad or mouse scroll wheel support.

**Steps to reproduce the behavior**
N/A

**Expected behavior**
Ability to scroll without the requirement of touchpad or mouse scroll wheels

**Screenshots**

Left Menu
![image](https://user-images.githubusercontent.com/8593673/64321351-393e0580-cf75-11e9-9b70-8e1965098d81.png)

Main Window:
![image](https://user-images.githubusercontent.com/8593673/64321403-4f4bc600-cf75-11e9-9004-bb74f234812a.png)

**Code snippets**
N/A

**System**
- Node.js version: v12
- NPM version: v6
- Strapi version: beta.15
- Database: MariaDB 10.4 
- Operating system: Linux Mint 19.3 Tina (Ubuntu 18.04 LTS)
- Browser: Chrome and Firefox latest stable

Tagging @davidkartuzinski per request as this was taken from Coding with Jason Livestream

**Comment by davidkartuzinski** [comment]
Thank you. @derrickmehaffy  Accessibility issue affects up to 15% of worldwide users and is an important issue to discuss.

**Comment by derrickmehaffy** [comment]
Fixed in beta.18 marking as closed
